src/printer/smt2/smt2_printer.cpp

```diff
@@ -1194,6 +1194,7 @@ std::string Smt2Printer::smtKindString(Kind k)
     case kind::INTS_DIVISION: return "div";
     case kind::INTS_MODULUS_TOTAL:
     case kind::INTS_MODULUS: return "mod";
+    case kind::INTS_LOG2: return "log2i";
```

Comment:
What naming schema are we following here? For power 2 it is: 
 
`case kind::POW2: return "int.pow2";`

So is there a reason this is not `int.log2`?

Reply:
fixed this one